.htaccess

```diff
@@ -1,9 +1,19 @@
+
+php_value display_errors off
+php_value display_startup_errors off
+
+<IfFile /var/www/html/_control/debug>
+  php_value display_startup_errors on
+  php_value display_errors stderr
+</IfFile>
+
```

Comment:
Some of the related PRs had
```
php_value include_path ".:/var/www/html/_includes/"
```

Is that not needed or handled elsewhere?

Reply:
Currently, that is handled in /resources/keyman-site.conf. We should probably consolidate (even consider moving away from .htaccess for performance reasons) but that can happen in a later PR